database/mssql/scripts/dbscripts/PSP_PIMS_S20_00/Test Data/03_DML_LEASE_TESTING.sql

```diff
@@ -0,0 +1,72 @@
+/* -----------------------------------------------------------------------------
```

Comment:
Thanks for adding this, I'm assuming that all of these scripts run successfully? (you may need to check the docker logs to confirm).

Reply:
In the future we probably want to do this in a separate PR.

Reply:
Yeah, it worked fine. Thats a good point, I was hoping that the changes being SQL would make it obvious, but will keep them separate next time